help/en/html/tools/logixng/reference/chapter8.shtml

```diff
@@ -62,14 +64,197 @@
       menu is used to <strong>Delete</strong> a local variable and has <strong>Move Up</strong> and
       <strong>Move Down</strong> options if there is more than one variable.</p>
 
+      <p>Click on <strong>OK</strong> to finish. The local variable will be added to the tree. See
+      the editor image below.</p>
+
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h3>Local variable types</h3>
+
       <div style="margin-left: 2em;">
         <a href="images/chapter8/local_variable_types.png"><img src=
-        "images/chapter8/local_variable_types.png" alt="Chapter 8 local variable types" width="458"
-        height="277"></a>
+        "images/chapter8/local_variable_types.png" alt="Chapter 8 local variable types" width="135"
+        height="188"></a>
       </div>
 
-      <p>Click on <strong>OK</strong> to finish. The local variable will be added to the tree. See
-      the image below.</p>
+      <dl>
+        <dt>None</dt>
+        <dd>Generic type which will change based on its content</dd>
+
+        <dt>Integer</dt>
+        <dd>A whole number such as 123</dd>
+
+        <dt>Floating number</dt>
+        <dd>A number such as 123.4</dd>
+
+        <dt>String</dt>
+        <dd>A series of characters, such as ABC123def</dd>
+
+        <dt>Array</dt>
+        <dd>A list of items.  This is implemented using the Java ArrayList.  This is similar to the
+        Python List.</dd>
+
+        <dt>Map</dt>
+        <dd>A list of kev/value items.  This is implemented using the Java HashMap.  This is similar
+        to the Python Dictionary.</dd>
+
+        <dt>Local Variable</dt>
+        <dd>Use the value from another local variable.</dd>
+
+        <dt>Memory</dt>
+        <dd>Use the value from a memory variable.</dd>
+
+        <dt>Reference</dt>
+        <dd>Use the value from an indirect reference.</dd>
+
+        <dt>Formula</dt>
+        <dd>Use the value from a formula</dd>
+
+      </dl>
+
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h5>Array<span class="since">since 4.25.8</span></h5>
+
+      <p>When an <strong>Array</strong> Local Variable is defined, it will default to empty.  Rows
+      can be added to the Array using the <strong>add</strong> method.  The rows can also be
+      defined by providing a value in the data column, in this example 5 rows for Array2.</p>
```

Comment:
`Array2` seems to be misspelled?

Reply:
I don't see a spelling error.

Reply:
I apologize. I didn't realize that Array2 referred to a local variable.